crates/bevy_ecs/src/reflect/world.rs

```diff
@@ -0,0 +1,194 @@
+use bevy_reflect::{Reflect, ReflectFnsTypeData, ReflectFromPtr};
+
+use crate::component::ComponentId;
+use crate::entity::Entity;
+use crate::prelude::{AppTypeRegistry, World};
+
+/// An extension trait for World for reflection related functions
+pub trait WorldExt {
```

Comment:
I like having these methods stored separately, but I don't really see the point of the trait extension method rather than just another impl block.